lib/rucio/client/baseclient.py

```diff
@@ -538,6 +539,49 @@ def __get_token_gss(self):
         LOG.debug('got new token')
         return True
 
+    def __get_token_saml(self):
+        """
+        Sends a request to get an auth token from the server and stores it as a class attribute. Uses saml authentication.
+
+        :returns: True if the token was successfully received. False otherwise.
+        """
+
+        headers = {'X-Rucio-Account': self.account}
+        userpass = {'username': self.creds['username'], 'password': self.creds['password']}
+        url = build_url(self.auth_host, path='auth/sso')
```

Comment:
maybe the endpoint should actually be called `auth/saml` ? the mental jump from sso to saml is not obvious for people not used to AAI